src/pages/studyView/StudyViewUtils.spec.ts

```diff
@@ -1860,11 +1852,23 @@ describe('StudyViewUtils', () => {
                 y: 1,
             });
             assert.deepEqual(
-                findSpot([['1', '1', ''], ['2', '2', '']], { w: 1, h: 1 }),
+                findSpot(
+                    [
+                        ['1', '1', ''],
+                        ['2', '2', ''],
+                    ],
+                    { w: 1, h: 1 }
+                ),
                 { x: 2, y: 0 }
             );
             assert.deepEqual(
-                findSpot([['1', '1', ''], ['2', '2', '']], { w: 2, h: 1 }),
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki